torch/_dynamo/variables/lists.py

```diff
@@ -25,27 +25,47 @@
 from .functions import UserFunctionVariable, UserMethodVariable
 
 
-def _listlike_contains_helper(items, search, tx, options):
+def _listlike_contains_helper(items, search, tx, options, check_hash_match=False):
     if search.is_python_constant():
-        result = any(
+        found = any(
```

Comment:
This path will probably fail if you are looking for a python constant on a listlike that just has tensors. Could you add a test?

Reply:
No, my point is that here you'll call `as_python_constant` on a tensor and it will raise an exception :D I think that you need to add an  `x.is_python_constant() and` in the condition

Reply:
Oh right. Makes sense.

Reply:
Fixed in: https://github.com/pytorch/pytorch/pull/111572